satellite/satellitedb/attribution_test.go

```diff
@@ -87,5 +88,21 @@ func TestUsers(t *testing.T) {
 			DefaultSegmentsSize: int64(100),
 		})
 		require.NoError(t, err)
```

Comment:
**Rational for code Changes:**
added test case to test the update bucket db function. Also changed the name from bucket_test to attribution_test.go as these test cases are more related to attribution feature.